MethodDungeonTools.lua

```diff
@@ -1629,14 +1629,17 @@ end
 
 function MethodDungeonTools:FormatEnemyHealth(amount)
 	amount = tonumber(amount)
-	if amount<1000 then return ""; end
-    if amount<10000 then return string.sub(amount,1,1).."k"; end --1k
-    if amount<100000 then return string.sub(amount,1,2).."k"; end --10k
-    if amount<1000000 then return string.sub(amount,1,3).."k"; end --100k
-    if amount<10000000 then return string.sub(amount,1,1).."."..string.sub(amount,2,3).."m"; end --1.11m
-    if amount<100000000 then return string.sub(amount,1,2).."."..string.sub(amount,3,4).."m"; end --11.11m
-    if amount<1000000000 then return string.sub(amount,1,3).."."..string.sub(amount,4,5).."m"; end --111.11m
-    return string.sub(amount,1,1).."."..string.sub(amount,2,3).."b"; --1.11b
+    if amount < 1e3 then
+        return ""
```

Comment:
Is there a good reason to not return an enemy health amount here? Couldn't we just return `amount`?

Reply:
The function that I based this one (that I use in my addon) returns floor(amount) but I changed it to match the current code.

Reply:
Perhaps @nnogga has a better explanation of why no health for low amounts? 

Not a deal breaker right now, but it may be more important after the BfA stat squish, when there is a higher possibility of valid mobs actually having sub-1k health pools.

Reply:
no reason for this - this is a remnant of the original function which showed absorbs for different units